dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/PatientResource.java

```
@@ -110,12 +102,42 @@ public Bundle patientSearch(@ApiParam(hidden = true)
             } else {
                 expandedMBI = String.format("%s|%s", DPCIdentifierSystem.MBI.getSystem(), patientMBI);
             }
-            return request
-                    .where(Patient.IDENTIFIER.exactly().identifier(expandedMBI))
-                    .execute();
+            query = query.where(Patient.IDENTIFIER.exactly().identifier(expandedMBI));
         }
 
-        return request.execute();
+        return query;
+    }
+
+    @GET
+    @FHIR
+    @Timed
+    @ExceptionMetered
+    @Authorizer
+    @ApiOperation(value = "Search for Patients", notes = "FHIR endpoint for searching for Patient resources." +
+            "<p> If Patient Identifier is provided, results will be filtered to match the given property")
+    @Override
+    public Bundle patientSearch(@ApiParam(hidden = true)
+                                @Auth OrganizationPrincipal organization,
+                                @ApiParam(value = "Patient MBI")
+                                @QueryParam(value = Patient.SP_IDENTIFIER) @NoHtml String patientMBI,
+                                @ApiParam(value = "Patients per page")
+                                @QueryParam(value = "_count") Integer count,
+                                @ApiParam(value = "Page number") // null means "do not paginate" for compatibility reasons
+                                @QueryParam(value = "_page") Integer page) {
+        var request = this.buildPatientSearchQuery(organization.getOrganization().getId(), patientMBI);
+        if (count == null && page != null) {
+            count = this.defaultPageSize;
+        }
+
+        if (count != null && count == 0) {
+            request.count(0).summaryMode(SummaryEnum.COUNT); // count gets omitted in request
+            return request.execute();
+        }
+        else if (count !=null && page != null && page >= 1) {
```

If they pass us a negative `_count` or `_page` can/should we throw an error and return a 4xx?